static/src/javascripts/projects/commercial/modules/header-bidding/prebid/bid-config.js

```diff
@@ -289,6 +291,19 @@ const getTripleLiftInventoryCode = (
     return '';
 };
 
+const getOzoneTargeting = (): { } => {
+    const lotameData: LotameData = getLotameData();
+
+    if (typeof lotameData !== 'undefined') {
```

Comment:
Is there any benefit of using `typeof var !== 'undefined'` ? I could not find a clear recommendation either way.

```suggestion
    if (lotameData) {
```

Reply:
I think it depends from the use case. In this instance, It's just more explicit that we are checking we have received a response form lotame's callback and have initialised the variable. In it's worst case it will be something like { ozoneLotameData: 'whatever',  ozoneLotameProfileId: whatever }. Without the `typeof var !== 'undefined'`, is checking additional things such as : null, empty string, 0, NaN, false which are not part of the use case in this instance.